src/adapter/scriptSkipper.ts

```diff
@@ -71,140 +91,154 @@ export class ScriptSkipper {
   }
 
   private _createRegexString(patterns: string[]): string {
+    // TODO this should use node-glob directly
```

Comment:
I'll do this

Reply:
We use minimatch already for some other stuff, you can grab that